### ISSUE TITLE
Negative amount are wrongly identified as 0

### ISSUE THREAD

**Issue by BoGnY** [finller/laravel-money#12]
Negative amount are wrongly identified as 0:

Expected:
```php
dump(\Brick\Money\Money::of('-1360', 'EUR'));

// Expected result:
Brick\Money\Money {#1838 ▼
  -amount: Brick\Math\BigDecimal {#1886 ▼
    -value: "-136000"
    -scale: 2
  }
  -currency: Brick\Money\Currency {#1868 ▼
    -currencyCode: "EUR"
    -numericCode: 978
    -name: "Euro"
    -defaultFractionDigits: 2
  }
  -context: Brick\Money\Context\DefaultContext {#1839}
}

```

Currently `MoneyParser()` try to identify amount type (float, int, string, etc...) and call `Money::of()`:
```php
dump(\Finller\Money\MoneyParser::of('-1360', 'EUR'));

// Currently result:
Brick\Money\Money {#1876 ▼
  -amount: Brick\Math\BigDecimal {#1625 ▼
    -value: "0" // <-- THE ERROR in the MoneyParser
    -scale: 2
  }
  -currency: Brick\Money\Currency {#1868 ▼
    -currencyCode: "EUR"
    -numericCode: 978
    -name: "Euro"
    -defaultFractionDigits: 2
  }
  -context: Brick\Money\Context\DefaultContext {#1865}
}

```

**Comment by BoGnY** [comment]
@QuentinGab I've identified the bug.. I'm doing a PR 👍🏻

**Comment by QuentinGab** [comment]
@BoGnY Thanks ! it is merged and tagged in v1.0.1
